lib/object.h

```diff
@@ -13,6 +13,40 @@ struct tpm2_loaded_object {
     tpm2_session *session;
 };
 
+typedef struct tpm_load_ctx tpm_load_ctx;
```

Comment:
I believe only object strucutures should go to object.h, not all the tool options as they do not belong here.